share/gitHooks/pre-push

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+
+# Run clang-format on all .cpp and .h files to be pushed
+for FILE in $(git diff --name-only origin/master |grep -E "\.h|\.cpp")
```

Comment:
This is referencing master instead of main.

Reply:
Thanks for the catch.